maat/tests/real_world.rs

```diff
@@ -191,14 +191,15 @@ async fn publish_storage_deals<Keypair>(
 
     for event in deal_result
         .events
-        .find::<storagext::runtime::market::events::DealPublished>()
+        .find::<storagext::runtime::market::events::DealsPublished>()
```

Comment:
Why not use `find_first` here, since it is only looking for a single occurrence?